tensorflow_addons/losses/triplet.py

```diff
@@ -79,6 +82,11 @@ def triplet_semihard_loss(
       y_pred: 2-D float `Tensor` of embedding vectors. Embeddings should
         be l2 normalized.
       margin: Float, margin term in the loss definition.
+      distance_metric: str or function, determines distance metric:
+                       "L_1" for l1-norm distance
+                       "L_2" for l2-norm distance
+                       "angular" for cosine similarity
+                        function for custom metric
```

Comment:
You probably want to give an example of custom function,

You can see the example here:

https://www.tensorflow.org/api_docs/python/tf/keras/callbacks/LearningRateScheduler